samples/boards/st/mco/boards/nucleo_f411re.overlay

```diff
@@ -0,0 +1,32 @@
+/* Enable the PLLI2s and set it as clock source for the MCO2 (with prescaler 5) */
+
+&plli2s {
+	div-m = <8>;
+	mul-n = <100>;
+	div-r = <2>;
+	clocks = <&clk_hse>;
+	status = "okay";
+};
+
+/* see RefMan RM0383 */
+&mco1 {
+	/* Select One of the line below for clock source */
+	clocks = <&rcc STM32_SRC_HSI MCO1_SEL(0)>;
+/*	clocks = <&rcc STM32_SRC_LSE MCO1_SEL(1)>; */
+/*	clocks = <&rcc STM32_SRC_HSE MCO1_SEL(2)>; */
+/*	clocks = <&rcc STM32_SRC_PLL_P MCO1_SEL(3)>; */
+	prescaler = <MCO1_PRE(4)>; /* set 0b100 in the MCO1PRE of the RCC CFGR */
+	pinctrl-0 = <&rcc_mco_1_pa8>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+#define MCO2_PRE_DIV_5 7
+
+&mco2 {
+	clocks = <&rcc STM32_SRC_PLLI2S_R MCO2_SEL(1)>;
+	prescaler = <MCO2_PRE(MCO2_PRE_DIV_5)>; /* set 0b111 in the MCO2PRE of the RCC CFGR */
```

Comment:
Usage of `MCO2_PRE_DIV_5` greatly increase usability of the property.
I think we should add as part of `include/zephyr/dt-bindings/clock/stm32yy_clock.h` files directly so that all users could benefit.

Reply:
adding MCO2_PRE_DIV_n for stm32XX_clock.h bindings files